src/utils/permissions.ts

```diff
@@ -0,0 +1,17 @@
+import { CommandInteraction, PermissionString, TextChannel, User } from 'discord.js';
+import { check } from './emotes.json';
+
+export function hasPermissions(int: CommandInteraction, permissions: PermissionString[]) {
+	const channel = int.channel as TextChannel;
+
+	const perms = permissions.map(perm => channel.permissionsFor(int.client.user as User)!.has(perm));
+	const hasPerms = !perms.includes(false);
+
+	const p = permissions.filter((perm, i) => perms[i] === false);
+	const missingP = p.map(per => `**\`${per}\`**`).join(' ');
+	if (!hasPerms) {
+		int.reply(`${check.red} É preciso das permissões ${missingP} para executar esse comando.`);
```

Comment:
So... this kinda confused me when reviewing `guess.ts`. I thought we weren't replying there.
I think this should be responsability of the caller (single-responsability principle again lol).

Reply:
Already, I guess it is fair enough to me LOL. Will refator.